charts/consul/templates/crd-gatewayclassconfigs-v1.yaml

```diff
@@ -0,0 +1,201 @@
+{{- if .Values.connectInject.enabled }}
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.12.1
+  labels:
+    app: {{ template "consul.name" . }}
+    chart: {{ template "consul.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: crd
+  name: gatewayclassconfigs.consul.hashicorp.com
+spec:
+  group: consul.hashicorp.com
+  names:
+    kind: GatewayClassConfig
+    listKind: GatewayClassConfigList
+    plural: gatewayclassconfigs
+    singular: gatewayclassconfig
+  scope: Cluster
+  versions:
+    - name: v1alpha1
```

Comment:
should this be v2beta1?

Reply:
This is the V1 `gatewayClassConfig`. It won't be named `external` like we have for the `Gateway` and `GatewayClass`, so to avoid naming issues the file is being renamed here.